templates/controllers/grid/users/reviewer/readReview.tpl

```diff
@@ -20,35 +20,56 @@
 	<input type="hidden" name="reviewAssignmentId" value="{$reviewAssignment->getId()|escape}" />
 	<input type="hidden" name="submissionId" value="{$reviewAssignment->getSubmissionId()|escape}" />
 	<input type="hidden" name="stageId" value="{$reviewAssignment->getStageId()|escape}" />
-	<p>{translate key="editor.review.readConfirmation"}</p>
+	{if $reviewAssignment->getUnconsidered() == $smarty.const.REVIEW_ASSIGNMENT_UNCONSIDERED}
+		<p>{translate key="editor.review.readConfirmation"}</p>
+	{elseif !$reviewAssignment->getDateCompleted()}
+		<p>{translate key="editor.review.reviewDetails.readConfirmation.description"}</p>
+	{/if}
 	<div id="reviewAssignment-{$reviewAssignment->getId()|escape}">
 		<h2>{$reviewAssignment->getReviewerFullName()|escape}</h2>
-		<span class="pkp_controllers_informationCenter_itemLastEvent">{$reviewAssignment->getDateCompleted()|date_format:$datetimeFormatShort}</span>
 
-		{if $reviewAssignment->getReviewFormId()}
-			{include file="reviewer/review/reviewFormResponse.tpl"}
-		{else}
-			<h3>{translate key="editor.review.reviewerComments"}</h3>
-			{assign var="contents" value=$reviewerComment->getComments()}
-			<span id="reviewContents">
-				{$contents|truncate:250|nl2br|strip_unsafe_html}
-				{if $contents|strlen > 250}<a href="javascript:$.noop();" class="showMore">{translate key="common.more"}</a>{/if}
-			</span>
-			{if $contents|strlen > 250}
-				<span class="hidden">
-					{$contents|nl2br|strip_unsafe_html} <a href="javascript:$.noop();" class="showLess">{translate key="common.less"}</a>
+		{if $reviewAssignment->getDateCompleted()}
+			<p>{translate key="common.completed.date" dateCompleted=$reviewAssignment->getDateCompleted()|date_format:$datetimeFormatShort}</p>
+
+			{if $reviewAssignment->getReviewFormId()}
+				{include file="reviewer/review/reviewFormResponse.tpl"}
+			{elseif $reviewerComment}
+				<h3>{translate key="editor.review.reviewerComments"}</h3>
+				{assign var="contents" value=$reviewerComment->getComments()}
+				<span id="reviewContents">
+					{$contents|truncate:250|nl2br|strip_unsafe_html}
+					{if $contents|strlen > 250}<a href="javascript:$.noop();" class="showMore">{translate key="common.more"}</a>{/if}
```

Comment:
Ah, I thought @NateWr would do it after the merge, with a solution with several lines... but I can do it if Nate tell me how :-) I think something like this could work:
```
width: 500px;
height: 100px;
overflow-y: hidden;
```
But it would be without "..." :-)
Nate what do you prefer or recommend?
Thanks!!!